iac-common/src/main/java/org/sonar/iac/common/parser/grammar/Punctuator.java

```diff
@@ -56,7 +56,8 @@ public enum Punctuator implements GrammarRuleKey {
   PERCENT_LCURLY_TILDE("%{~"),
   TILDE_RCURLY("~}"),
   DOLLAR("$"),
-  APOSTROPHE("'");
+  APOSTROPHE("'"),
```

Comment:
I don't see any use of a single `APOSTROPHE` if I'm not mistaken. Can be deleted IMO.

Reply:
It's used in `STRING_COMPLETE`, `INTERPOLATED_STRING_LEFT_PIECE`, and other places, but not in this PR.
Here the change is from semicolon to colon to add new Punctuators.

Reply:
Ah I missed that during the review.